src/Controls/StateControl.js

```diff
@@ -73,6 +73,27 @@ function stateToTrigger(state) {
 }
 
 
+function stateToEventType(stateControls, state, suffix = '') {
+    switch (state) {
+        case stateControls.ORBIT:
+            return `rotate${suffix}`;
+        case stateControls.MOVE_GLOBE:
+            return `drag${suffix}`;
+        case stateControls.DOLLY:
+            return `dolly${suffix}`;
+        case stateControls.PAN:
+            return `pan${suffix}`;
+        case stateControls.PANORAMIC:
+            return `panoramic${suffix}`;
+        case stateControls.TRAVEL_IN:
+            return 'travel_in';
+        case stateControls.TRAVEL_OUT:
+            return 'travel_out';
+        default:
+    }
+}
```

Comment:
could you define the type in state?

Reply:
I added a `_event` property in each state, and removed this method.